daml/splice-amulet/daml/Splice/Amulet.daml

```diff
@@ -24,20 +24,22 @@ import Splice.Types
 import Splice.Util
 
 data AmuletExpireSummary = AmuletExpireSummary with
-  owner : Party
-  round: Round -- ^ Round for which this expiry was registered.
-  changeToInitialAmountAsOfRoundZero : Decimal
-  -- The change to the total balance introduced by a amulet expiry, normalized to round zero, i.e.,
-  -- a amulet created in round 3 is treated as a amulet created in round 0 with a higher initial amount.
-  changeToHoldingFeesRate : Decimal
-    -- ^ The change of total holding fees introduced by a amulet expiry.
+    owner : Party
+    round: Round -- ^ Round for which this expiry was registered.
+    changeToInitialAmountAsOfRoundZero : Decimal
+    -- The change to the total balance introduced by a amulet expiry, normalized to round zero, i.e.,
+    -- a amulet created in round 3 is treated as a amulet created in round 0 with a higher initial amount.
+    changeToHoldingFeesRate : Decimal
+      -- ^ The change of total holding fees introduced by a amulet expiry.
+  deriving (Eq, Show)
```

Comment:
note: I only added these instead of adding them for all choice `Result` types. The latter seemed to bloat the `splice-amulet.dalf` file quite extensively for little gain.